pytest_bdd/scenario.py

```diff
@@ -330,6 +309,20 @@ def scenario(feature_name, scenario_name, encoding="utf-8", example_converters=N
     )
 
 
+def get_base_dir(caller_module):
+    default_base_dir = os.path.dirname(caller_module.__file__)
+    return get_from_ini('bdd_feature_base_dir', default_base_dir)
+
+
+def get_strict_gherkin():
+    return get_from_ini('bdd_strict_gherkin', True)
+
+
+def get_from_ini(key, default):
```

Comment:
I'm not sure if we need this function just to fallback to a default. I'd rather declare the default in the pytest.config.addini(), and handle specific fallback behavior only where needed (in this case, in the get_base_dir function).

Reply:
For `bdd_strict_gherkin` I already declared the default of `addini` call.
I'll remove `get_from_ini` call for `strict_gherkin`.